lisa/util/shell.py

```diff
@@ -161,42 +160,37 @@ def try_connect(connection_info: ConnectionInfo, ssh_timeout: int = 300) -> Any:
                 key_filename=connection_info.private_key_file,
                 banner_timeout=10,
             )
+
+            stdin, stdout, _ = paramiko_client.exec_command("cmd\n")
```

Comment:
Can you move this out to a separated PR? I need more time to look at it. We should make PR small to easy to review and proceed.

Reply:
Removed commit from this PR